pkgs/development/python-modules/kiss-headers/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, buildPythonPackage, fetchFromGitHub, python }:
+
+buildPythonPackage rec {
+  pname = "kiss-headers";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Ousret";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-/eTRyxFyAKQMzE/JjdoEN3w0lRiaIJcsJHTWV8M0CYQ=";
+  };
+
```

Comment:
We don't need requests? https://github.com/Ousret/kiss-headers/blob/master/requirements.txt

Reply:
Added to the propagated builds list